game-fallout4/index.js

```diff
@@ -93,11 +113,16 @@ function main(context) {
     ],
     requiresLauncher,
     environment: {
-      SteamAPPId: '377160',     
+      SteamAPPId: STEAM_ID,
+      GogAPPId: GOG_ID,
```

Comment:
As an FYI - the environment array represents environment variables used when running the tool/game executable. As far as I know, only Steam actually uses this when running a game, specifically the "SteamAppId" variable, all other properties are kind of redundant, but I guess it's not a big deal.